scripts/get_previous_timestamp.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import requests
+import json
+
+def parse_arguments():
+    """ parse command line arguments """
+    parser = argparse.ArgumentParser(description="Get workflow information")
+    parser.add_argument(
+        "-token",
+        required=True,
+        type=str,
+        help="Github access token",
+    )
+    parser.add_argument(
+        "-rid",
+        "--run-id",
+        required=True,
+        type=str,
+        help="Github access token",
```

Comment:
[here](https://github.com/ewlu/riscv-gnu-toolchain/actions/runs/6385527345/job/17330453934#step:4:134) we can see that the current run is immediately included in the api call. As of right now, I'm unsure if this will always be the case so I have the run id to ensure that we don't skip a scheduled run

Reply:
Ah sorry - I meant to edit this before submitting the review. I meant to update the help message

Reply:
Ah that makes a lot more sense with if statement